common/src/main/java/io/pravega/common/hash/HashHelper.java

```diff
@@ -54,6 +56,10 @@ public int hash(byte[] array, int offset, int length) {
         return hash.hashBytes(array, offset, length).asInt();
     }
 
+    public byte[] numToBytes(int num) {
```

Comment:
It is only used by a test. It is marked as not covered because the test is in a different package. 
If we want to remove it I could move it into the test itself or mark it as testOnly. But I think it would be OK if some future code used it outside of a test.

Reply:
This is only used by a tests. I have marked it as VisableForTesting.